core/store/orm/schema.go

```diff
@@ -54,7 +54,8 @@ type ConfigSchema struct {
 	EthHeadTrackerSamplingInterval            time.Duration   `env:"ETH_HEAD_TRACKER_SAMPLING_INTERVAL" default:"1s"`
 	EthLogBackfillBatchSize                   uint32          `env:"ETH_LOG_BACKFILL_BATCH_SIZE" default:"100"`
 	EthMaxGasPriceWei                         big.Int         `env:"ETH_MAX_GAS_PRICE_WEI"`
-	EthMaxUnconfirmedTransactions             uint64          `env:"ETH_MAX_UNCONFIRMED_TRANSACTIONS" default:"500"`
```

Comment:
should perhaps at first remain here but result in a warning asking to migrate to ETH_MAX_QUEUED_TRANSACTIONS ?

Reply:
I considered that... the issue is that node operators don't actually know what this option does and many have erroneously set it to something higher without understanding why.

This PR also changes the meaning of it somewhat, so I think it's OK to force everyone back to the default here unless they explicitly change it.

Reply:
reasonable, yes